webapp/src/components/ImageFieldEditor.tsx

```diff
@@ -197,7 +227,77 @@ export class ImageFieldEditor<U extends pxt.Asset> extends React.Component<Image
         this.galleryAssets = getAssets(true, this.asset.type);
     }
 
-    protected filterAssets(assets: pxt.Asset[], type: pxt.AssetType = this.asset?.type, isGallery = false, useTags?: boolean) {
+    protected getAvailableTags(filterAssets: pxt.Asset[]) {
+        let collectedTags: string[] = [];
+        // Pixel Art Categories -- Add new categories here!
+        // lf("People")
+        // lf("Animals")
+        // lf("Food")
+        // lf("Dungeon")
+        // lf("Forest")
+        // lf("Space")
+        // lf("Aquatic")
+
+        if (this.galleryAssets) {
+            const specialTags = ["tile", "dialog", "background"];
```

Comment:
maybe this should be passed in to "getAvailableTags" as a "tags to omit" param or something? i could picture a case where having these in the filter panel might be useful